portal-api/src/scheduler/jobs/index.js

```diff
@@ -1,7 +1,8 @@
 const sendReportDueEmailsJob = require('./send-report-due-emails-job');
```

Comment:
It's a nit pick and I think we're fine to change but just on naming here:
ReportDue is kind of true for all of them, I think this would be sendReportSubmissionPeriodEndEmailsJob or something instead.
Feel free to just resolve this as I think the rest of this section of code explains all the functionality well